backend/app/utils/credentials/base_credential_manager.py

```
@@ -0,0 +1,52 @@
+
+from omegaconf import DictConfig
+from registrable import Registrable
+
+from app.data_layer.data_models.credential_model import (
+    CredentialInput,
+    CredentialOutput,
+)
+
+
+@abstractmethod
+class CredentialManager[CI: CredentialInput, CO: CredentialOutput](Registrable, ABC):
```

Pylint (3.3.7)

[error] Parsing failed: 'invalid syntax (credentials.base_credential_manager, line 13)'

(E0001)